### PR TITLE
Accept more captcha formats

### PR DESCRIPTION
**Pull request by jowlo**
fix #2603

### CODE REVIEW

**Comment by charlag** [comment]
Can we extract it and add tests please?

**Comment by jowlo** [comment]
> Can we extract it and add tests please?

Sure, just did. I created a new test file`SignupFormTest.js`. It felt overkill but i didn't see a better place to put it.

File: src/api/main/SignupForm.js [hunk 1]
```diff
@@ -177,6 +177,20 @@ function signup(mailAddress: string, pw: string, registrationCode: string, isBus
 	})
 }
 
+/**
+ * Accepts multiple formats for a time of day and always returns 12h-format with leading zeros.
+ * @param captchaInput
+ * @returns {string} HH:MM if parsed, empty string otherwise
+ */
+export function parseCaptchaInput(captchaInput: string): string {
```
Comment (charlag) [inline src/api/main/SignupForm.js:185]: I would rather return `null` so that it's visible on type level that there are two cases

**Comment by jowlo** [comment]
Yeah, i was unsure which one would be nicer. But you are right...

**Comment by charlag** [comment]
Thanks, jsdoc is out of date now but I can change it when I merge it

**Comment by jowlo** [comment]
Oh, yes, thanks. I also renamed the test file to end in `.js`  *facepalm*

**Review by charlag — APPROVED** [review]

**Pull request merged**